resources/views/widgets/settings/availability-map.blade.php

```diff
@@ -56,17 +56,17 @@
 
     <div class="form-group">
         <label for="device_group-{{ $id }}" class="control-label">@lang('Device group')</label>
-        <select class="form-control" name="device_group" id="device_group-{{ $id }}"  data-placeholder="@lang('All Devices')">
+        <select class="form-control" name="device_group" id="device_group-{{ $id }}" data-placeholder="@lang('All Device groups')">
             @if($device_group)
-                <option value="{{ $device_group->id }}" selected> {{ $device_group->name }} </option>
+                <option value="{{ $device_group->id }}" selected>{{ $device_group->name }}</option>
             @endif
         </select>
     </div>
 @endsection
 
 @section('javascript')
     <script type="text/javascript">
-        init_select2('#device_group-{{ $id }}', 'device-group', {}, '{{ $device_group ? $device_group->id : '' }}');
+        init_select2('#device_group-{{ $id }}', 'device-group', {});
```

Comment:
When you remove this, is the current device group pre-selected?

Reply:
Yes, and now it shows the name from the `<option>' instead of the id

Reply:
Oh yeah, you have to pass an object there with name and id (then you can avoid the select in html).  I'm not sure which one is better...